public/pages/games/mistureba.html

```diff
@@ -6,122 +7,241 @@
+              </svg>
+            </button>
+            <button type="button"  class="btn btn-outline-success" onclick="tip()" >
+             <strong>	 
+               OK
+             </strong>
+           </button>
+          </div>
+        </div> 
+      </div>   
+    </aside>
+  </main>
+ </div> 
+  
 
-      </div>
-      <hr>
-    </div>
-    <div class="result1">
-      <div class="result">
-        <h4>PONTOS TOTAIS</h4>
-        <h4>0</h4>
 
-      </div>
 
-    </div>
-    <hr>
-  </main>
 
-  <!---Começo Rodapé-->
-  <footer class="bg-light text-center text-lg-start">
-    <!-- Copyright -->
-    <div class="text-center p-3" style="background-color: rgba(0, 0, 0, 0.2)">
-      © 2022 Copyright:
-      <a class="text-dark" href="#">RecodePro 2022 - Módulo 2 Atividade prática Squad 61.</a>
+<!---INICIO RODAPÉ-->
+<footer class=" footer-container py-5 conteiner-fluid row">
+  <div class="col-6 col-md">
+    <img width= "200px + 10vw" src="../../../public/assets/images/logo.png">
+  </div>
+  <div class="col-6 col-md">
+        <h5 class="title text-secondary">
+          Termos:
+        </h5>
+      <ul class="list-unstyled text-small">
+        <li><a class="text-muted" href="https://recodepro.org.br/" target="_blank">Recode Pro</a></li>
+        <li><a class="text-muted" href="" target="_blank">Termos e Condições</a></li>
+        <li><a class="text-muted" href=""target="_blank">Política de Privacidade</a></li>
+      </ul>
+    </div>
+    <div class="col-6 col-md">
+        <h5 class="title text-secondary">
+          Canais de Atendimento:
+        </h5>
+        <p>De segunda á sexta-feira: 08:00 ás 20:00</p>
+        <img class="img-fluid" src="../../../public/assets/images/contato.png"> +55 00 0000-0000 </a>
+        <p><img src="../../../public/assets/images/email.png"> recreio.org@gmail.com </a> </p>
     </div>
-    <!-- Copyright -->
+    <div class="col-6 col-md">
+        <h5  class="title text-secondary">
+          Follow us:
+        </h5>
+        <ul class="list-unstyled text-small">
+        <a href="https://github.com/Squad-61/" target="_blank"> <img src="../../../public/assets/images/github.png"> </a>
+        <a href="https://www.instagram.com/" target="_blank"> <img src="../../../public/assets/images/instagram.png"> </a>
+        <a href="https://pt-br.facebook.com//" target="_blank"> <img src="../../../public/assets/images/facebook.png"> </a>
+        <a href="https://br.linkedin.com/" target="_blank"> <img src="../../../public/assets/images/linkedIn.png"> </a>
+        <a href="https://www.youtube.com/" target="_blank"> <img src="../../../public/assets/images/youtube.png"> </a>
+       </ul>
+    </div> 
+  <hr>
+  <section class ="d-flex justify-content-center align-items-center">
+    &copy: Squad 61.
+    <img width="60px + 15vw" src="../../../public/assets/images/logosquad.png">
```

Comment:
Sugestão de correção:

```suggestion
    <img class="img-fluid" src="../../assets/images/logosquad.png">
```